app/javascript/packages/document-capture/components/file-input.tsx

```diff
@@ -307,20 +308,31 @@ function FileInput(props: FileInputProps, ref: ForwardedRef<any>) {
           {hint}
         </span>
       )}
+
       <StatusMessage status={Status.ERROR} id={errorId}>
         {shownErrorMessage}
       </StatusMessage>
-      <StatusMessage
-        id={successId}
-        status={Status.SUCCESS}
+
+      <span
         className={
```

Comment:
We probably don't need the `className` on the `StatusMessage` below anymore with it being assigned here now.